proposals/0064-remote-control.md

```diff
@@ -0,0 +1,822 @@
+# SDL Remote Control
+
+* Proposal: [SDL-NNNN](NNNN-remote-control.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+SmartDeviceLink provides a framework that connects in-vehicle infotainment system to mobile phone applications. SDL  enables a driver to interact with their mobile phone applications using common in-vehicle interfaces such as a touch screen display, embedded voice recognition, steering wheel controls and various vehicle knobs and buttons. Although SDL provides some RPCs to allow mobile applications to obtain some vehicle status information, it does not allow a mobile application to change the vehicle settings, i.e. to control the vehicle. SDL remote control (or previous known as reverse SDL) provides the ability for mobile applications to control certain settings of the vehicle, such as, radio and climate.
+
+## Motivation
+
+There are many cases that mobile application developers ask for new functions of SDL. For example, 
+
+- A radio application wants to use the in-vehicle radio. It needs the RPC function to select the radio band (AM/FM/XM/HD/DAB), tune the radio frequency or change the radio station, as well as obtain general radio information for decision making. 
+
+- A climate control application needs to turn on the AC, control the air circulation mode, change the fan speed and set the desired cabin temperature. 
+
+- A user profile application wants to remember user's favorite settings and apply it later automatically when the users get into the same/another vehicle.
+
+- A navigation application want to set the applications display mode same as the vehicle's display mode and vice versa.
+
+The common problem is that a mobile application need the ability to control certain settings of the vehicle. This proposal (SDL remote control or SDL-RC) tries to address this problem. This is not a new proposal. This document describe the design of current implementation of SDL-RC feature branch.
```

Comment:
Also, "This is not a new proposal." is a bit misleading, as this is a new proposal in the SDL Evolution Process.  Can you please remove?